src/GovUk.Education.ExploreEducationStatistics.Common/Services/FileStoragePathUtils.cs

```diff
@@ -7,6 +7,11 @@ namespace GovUk.Education.ExploreEducationStatistics.Common.Services
 {
     public static class FileStoragePathUtils
     {
+        public const string PublicContentDataBlocksDirectory = "data-blocks";
```

Comment:
Now we have this, you can use it in PrivateContentDataBlockParentPath:
```
        public static string PrivateContentDataBlockParentPath(
            Guid publicationId,
            Guid releaseId)
        {
            return $"{PrivateContentReleaseParentPath(publicationId, releaseId)}/data-blocks";
        }
```

Reply:
Done!